src/index.css

```diff
@@ -56,7 +56,7 @@ body {
 .grid {
   position: absolute;
   width: 300%;
-  left: -75%;
+  left: -100%;
```

Comment:
It was a little off-center on big screens: 
![image](https://user-images.githubusercontent.com/49658988/174146485-e6a2b1c5-2ded-4f81-bc65-e5c82343104d.png)
Now: 
![image](https://user-images.githubusercontent.com/49658988/174146509-67011328-bf9e-4650-a633-05c1f051b9da.png)